src/components/Pagination.js

```diff
@@ -0,0 +1,41 @@
+const Pagination = (props) => {
+  const arr = ['1', '2', '3', '4', '5'];
+  const filters = document.querySelectorAll('.sidebar__link');
+  const pags = document.querySelectorAll('.pag__link');
+  const filterUrl = 'v2.1/films/search-by-filters?genre=';
+  return (
+    <div class="pagination">
+      {arr.map((i) => {
+        return (
+          <a
+            href="#"
+            className="pag__link"
+            data-page={i}
+            onClick={(e) => {
+              pags.forEach((i) => {
```

Comment:
Cool that you implement pagination by yourself. But `forEach` inside `forEach` seems to have very big complexity. And I'm not sure that it's unnecessary here. Maybe you have ideas how to simplify it?